Weatherman.py

```diff
@@ -0,0 +1,247 @@
+import os
+import sys
+
+
+def make_temperature_lists(file_name):
+    with open(file_name, 'r') as f:
+        min_temperature = 0
+        file_size = reading_file(file_name)
+        date_list = []
+        max_temperature_list = []
+        make_list = []
+        min_temperature_list = []
+        mean_list_humidity = []
+        file_contents = []
+        file_contents = f.readline()
+        i = 1
+        while i < file_size:
+            file_contents = f.readline()
+            make_list = file_contents.split(',')
+            max_temperature_list.append(make_list[1])
+            min_temperature_list.append(make_list[3])
+            mean_list_humidity.append(make_list[9])
+            date_list.append(make_list[0])
+            i += 1
+    return max_temperature_list, min_temperature_list, mean_list_humidity, date_list
+
+
+def reading_file(file_name):
+    file = open(file_name, "r")
+    Counter = 0
+    # Reading from file
+    Content = file.read()
+    CoList = Content.split("\n")
```

Comment:
in python don't user camel notion use `_` in variable name all character  must be small